go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/giantswarm/metrics-server-app
 
-go 1.14
+go 1.15
 
 require (
-	github.com/giantswarm/apiextensions/v2 v2.0.1 // indirect
-	github.com/giantswarm/appcatalog v0.2.7
+	github.com/giantswarm/apptest v0.4.1-0.20201030140104-d7a74a9bb5a5
```

Comment:
Needs https://github.com/giantswarm/apptest/pull/12

The dependencies get much simpler and we should be able to use dependabot for updating the apptest library.